tests/filters.test.js

```diff
@@ -41,5 +41,99 @@ describe(__filename, () => {
       expect(betterSafe(null)).toEqual(undefined);
       expect(betterSafe('')).toEqual(undefined);
     });
+
+    it('removes meta tags by default', () => {
+      const markup =
+        '<meta content="summary_large_image" name="twitter:card" some="attr">';
+      const betterSafe = makeBetterSafe({
+        markAsSafe(content) {
+          return content;
+        },
+      });
+
+      expect(betterSafe(markup)).toEqual('');
+    });
+
+    it('allows meta tags when isHeadMarkup is true', () => {
+      const markup =
+        '<meta content="summary_large_image" name="twitter:card" some="attr">';
+      const betterSafe = makeBetterSafe({
+        markAsSafe(content) {
+          return content;
+        },
+      });
+
+      expect(betterSafe(markup, { isHeadMarkup: true })).toEqual(
+        '<meta content="summary_large_image" name="twitter:card">'
+      );
+    });
+
+    it('accepts no tags when isHeadMarkup is true', () => {
+      const markup = '<!-- no content -->';
+      const betterSafe = makeBetterSafe({
+        markAsSafe(content) {
+          return content;
+        },
+      });
+
+      expect(betterSafe(markup, { isHeadMarkup: true })).toEqual('');
+    });
+
+    it('accepts property values starting with "og:"', () => {
+      const markup =
+        '<meta content="Mozilla Add-ons Blog" property="og:site_name">';
+      const betterSafe = makeBetterSafe({
+        markAsSafe(content) {
+          return content;
+        },
+      });
+
+      expect(betterSafe(markup, { isHeadMarkup: true })).toContain(
+        'property="og:site_name"'
+      );
+    });
+
+    it('allows link tags when isHeadMarkup is true', () => {
+      // The `content` attribute is only valid for `meta` tags so it should be
+      // removed on `link` tags.
+      const markup =
+        '<link rel="canonical" href="some-url" content="invalid" />';
+      const betterSafe = makeBetterSafe({
+        markAsSafe(content) {
+          return content;
+        },
+      });
+
+      expect(betterSafe(markup, { isHeadMarkup: true })).toEqual(
+        '<link rel="canonical" href="some-url">'
+      );
+    });
+
+    it('allows json+ld scripts (only) when isHeadMarkup is true', () => {
+      const markup =
+        '<script type="application/ld+json">{"@context":"https://schema.org"}</script>';
```

Comment:
What would happen if there's a second type attr set to `application/javascript`?